distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1718,10 +1719,13 @@ type Table
        GROUP Standard.Base.Calculations
        ICON sigma
 
-       Aggregates the rows in a table using any `Group_By` entries in columns.
-       The columns argument specifies which additional aggregations to perform and to return.
+       Aggregates the rows in a table using `group_by` columns.
+       The columns argument specifies which additional aggregations to perform
+       and to return.
```

Comment:
Btw. from this description I wouldn't know that I can do `aggregate` without `group_by` to get totals, i.e.
```
table.aggregate group_by=[] columns=[Aggregate_Column.Sum "Sale Price"]
```
to get one row containing total sales in a table.